src/FSharp.Control.TaskSeq/TaskSeq.fs

```diff
@@ -11,10 +11,15 @@ module TaskSeq =
     // Just for convenience
     module Internal = TaskSeqInternal
 
-    let empty<'T> = taskSeq {
-        for c: 'T in [] do
-            yield c
-    }
+    let empty<'T> =
```

Comment:
This is a good improvement. Can you please move this to `TaskSeqInternal`? That's where all the "larger than one or two lines" implementations go.

Reply:
Sure I can, but I wonder what's wrong on having a large file with implementations?
If we want a short one, we have already the fsi file IMHO having a large file, specially taking into account that we have a fsi file, is more that fine. Otherwise we'll have triplication.

Reply:
>  what's wrong on having a large file with implementations?

Nothing. It’s a personal preference. Ease of reading, clear separation of multi purpose implementation from single purpose wrappers.

I find (very) large implementation files hard to navigate. But there’s something to say for either approach. In this project I chose simplicity (smaller files with some logical, somewhat arbitrary separation) over one big implementation file (like in AsyncSeq). Both have pros and cons, but in the end, you choose a direction and stick to it, or it becomes messy.